cache/lru_cache.h

```diff
@@ -181,10 +182,15 @@ class LRUHandleTable {
 
   void Resize();
 
+  // Number of hash bits (upper because lower bits used for sharding)
+  // used for table index. Length == 1 << length_bits_
+  int length_bits_;
```

Comment:
uint32_t?

Reply:
For a shift amount, there's no signed/unsigned pollution (with decent compilers, `1U << some_int` is unsigned) and the values are small, so under Google C++ style, the default choice is `int`.